packages/react-scripts/template/README.md

```diff
@@ -566,6 +566,7 @@ Import Bootstrap CSS and optionally Bootstrap theme CSS in the ```src/index.js``
 import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap/dist/css/bootstrap-theme.css';
 ```
+Ensure the bootstrap css is imported before the rest of your app (i.e. before `import Routes from './routes';`) so that your styles are loaded afterwards.
```

Comment:
Maybe we can try to explain it simply. Something like:

Be mindful that the order of CSS matters. Typically, the last imported rule overrides previous rule(s).
If you would like to override some Bootstrap styles, make sure you import the Bootstrap CSS before the rest of your application.